schemas/data_catalogue/B18/current.json

```diff
@@ -0,0 +1,121 @@
+{
+    "$id": "/data_catalogue/B18/1.0.0",
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "title": "data_catalogue/B18",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "detectors": { 
+            "title": "Detectors",
+            "description": "The detector used for the scan, and the detection mode used",
+            "type": "array",
+            "items": {
```

Comment:
We have an issue that object arrays are hard to support with the bulk upload. We will need to talk to B18, but can we potentially change how we do this? For example could we have detectors as bools to say whether they are used or not, and a `detection_mode` for each?